pkgs/applications/graphics/inkscape/extensions/applytransforms/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+}:
+
+stdenv.mkDerivation {
+  pname = "inkscape-applytransforms";
+  version = "0.0.0+unstable=2021-05-11";
```

Comment:
```suggestion
  version = "unstable-2021-05-11";
```

Reply:
why? your suggestion is not the format recommended in the manual (https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming), and AFAICS also not the format that the discussion at https://discourse.nixos.org/t/brainstorming-for-rfc-pname-and-version/12873/83 is converging towards. I don't see how it is superior to the version from the PR.

Reply:
Until there is a consensus, I would like to use a format that can be easily converted to whatever format automatically, when a consensus is established. That requires including more information than the current manual suggests.

Reply:
I have read that manual section 10 times and this is what is written in it.

Reply:
As written, it can be also interpreted as `pname = "inkscape-applytransforms-unstable"; version = "2021-05-11"`. In fact that is the correct interpretation according to Eelco IIRC. But at this moment, the section does not know about `pname` and thus is ambiguously written.